### PR TITLE
[zh] Convert format for kubeadm-upgrade-npp page

### PR DESCRIPTION
**Pull request by tengqm**
This PR converts the EOL style from DOS to UNIX.

### CODE REVIEW

File: content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_preflight.md [hunk 1]
```diff
@@ -1,85 +1,75 @@
...
+</tr>
+
+<tr>
+<td colspan="2">--ignore-preflight-errors strings</td>
+</tr>
+<tr>
+<!-- 
+<td></td><td style="line-height: 130%; word-wrap: break-word;">A list of checks whose errors will be shown as warnings. Example: 'IsPrivilegedUser,Swap'. Value 'all' ignores errors from all checks.</td> 
+-->
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>错误将显示为警告的检查清单。示例：'IsPrivilegedUser,Swap'。值为'all'表示忽略所有检查的错误。</p></td>
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_preflight.md:47]:
```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>错误将显示为警告的检查清单。示例：“IsPrivilegedUser,Swap”。值为 “all” 表示忽略所有检查的错误。</p></td>
```

File: content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_preflight.md [hunk 2]
```diff
@@ -1,85 +1,75 @@
...
+<col span="1" />
+</colgroup>
+<tbody>
+
+<tr>
+<td colspan="2">--rootfs string</td>
+</tr>
+<tr>
+<!-- <td></td><td style="line-height: 130%; word-wrap: break-word;">[EXPERIMENTAL] The path to the 'real' host root filesystem.</td> -->
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>[实验] 到'真实'主机根文件系统的路径。</p></td>
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_preflight.md:70]:
```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>[实验]到“真实”主机根文件系统的路径。</p></td>
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Review by Sea-n — COMMENTED** [review]
/approve

**Pull request merged**